dynamo.js

```diff
@@ -0,0 +1,61 @@
+import {DynamoDB} from "@aws-sdk/client-dynamodb";
+
+const client = new DynamoDB({endpoint: "http://localhost:8000"});
+
+export class Dynamo {
+    async addUserToOnlineList(id, username) {
+        const params = {
+            TableName: "OnlineUsers",
+            Item: {
+                ID: {N: id},
```

Comment:
I believe that the agreed upon property name in the architecture docs was 'id'. This should be reflected in the other usages of this property. Also if we are going to be saving uuids in our **id**s which are strings, why are defining them as numbers?